dotCMS/src/main/java/com/dotmarketing/db/DBPropertiesDataSourceStrategy.java

```diff
@@ -61,7 +61,7 @@ public DataSource apply() {
                 throw new FileNotFoundException("DB properties file not found");
             }
             return new HikariDataSource(getHikariConfig());
-        } catch (IOException e) {
+        } catch (Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5603107)